### PR TITLE
Feat/price utils

### PR DESCRIPTION
**Pull request by mkermani144**
Related: #17

### CODE REVIEW

File: src/arbitrategy.ts [hunk 1]
```diff
@@ -37,7 +40,10 @@ const adaErgArbitrategy: Arbitrategy = [
           "ae97c5eccd59a065cd973a8d6afb8bb79f9cc70368a7dcdf73aaeab1cedf6f6b",
       },
     ],
-    primaryAsset: "TBD",
+    primaryAsset: {
+      coingeckoId: "cardano",
```
Comment (fatemeh-ra) [inline src/arbitrategy.ts:44]: primary asset of this chain is Erg

File: src/arbitrategy.ts [hunk 2]
```diff
@@ -78,7 +87,10 @@ const adaRsnArbitrategy: Arbitrategy = [
           "29f45df2736a7c7dd790b682fb15429961d7c1ae92e05c72a4f65d53fc0c47d7",
       },
     ],
-    primaryAsset: "TBD",
+    primaryAsset: {
+      coingeckoId: "cardano",
```
Comment (fatemeh-ra) [inline src/arbitrategy.ts:91]: primary asset of this chain is RSN

File: src/utils.ts [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+export const asset2usd = async (asset: PrimaryAsset, amount: number) => {
+  const assetValue = await fetchPrice(asset.coingeckoId);
+
+  return (assetValue * amount) / asset.decimals;
+};
+
+export const usd2asset = async (asset: PrimaryAsset, usd: number) => {
+  const assetValue = await fetchPrice(asset.coingeckoId);
+
+  return (usd * asset.decimals) / assetValue;
```
Comment (fatemeh-ra) [inline src/utils.ts:26]:
```suggestion
  return (usd * (10**asset.decimals)) / assetValue;
```

File: src/utils.ts [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
-  usd: number
-): Promise<AmountWithDecimal> => {
-  return Promise.resolve({
-    amount: BigInt(0),
-    decimals: 0,
-  });
+export const asset2usd = async (asset: PrimaryAsset, amount: number) => {
+  const assetValue = await fetchPrice(asset.coingeckoId);
+
+  return (assetValue * amount) / asset.decimals;
```
Comment (fatemeh-ra) [inline src/utils.ts:20]:
```suggestion
  return (assetValue * amount) / (10**asset.decimals);
```

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 4 above)
Comment (mkermani144) [reply src/utils.ts]: done

File: src/utils.ts (hunk 3 above)
Comment (mkermani144) [reply src/utils.ts]: done

**Review by fatemeh-ra — APPROVED** [review]

**Pull request merged**